flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -426,6 +426,49 @@ class Table(
   }
 
   /**
+    * Set minus between two [[Table]]s. Similar to an SQL EXCEPT ALL.
+    * The fields of the two minus operands must fully overlap.
+    *
+    * Note: Both tables must be bound to the same [[TableEnvironment]].
+    *
+    * Example:
+    *
+    * {{{
+    *   left.minusAll(right)
+    * }}}
+    */
+  def minusAll(right: Table): Table = {
+    // check that right table belongs to the same TableEnvironment
+    if (right.tableEnv != this.tableEnv) {
+      throw new ValidationException("Only tables from the same TableEnvironment can be unioned.")
+    }
+    new Table(tableEnv, SetMinus(logicalPlan, right.logicalPlan, true).validate(tableEnv))
+  }
+
+  /**
+    * Perform set minus between [[Table]]s with duplicate records removed.
```

Comment:
Please describe the semantics of minus all in more detail:
- records of the first table are returned
- how many identical records are returned